cpp/bench/sg/linkage.cu

```diff
@@ -77,7 +77,7 @@ class Linkage : public BlobsFixture<D> {
  private:
   int* labels;
   int* out_children;
-  raft::hierarchy::linkage_output<int> out_arrs;
+  cuvs::cluster::agglomerative::single_linkage_output<int> out_arrs;
```

Comment:
This file has includes this header now that should not be needed anymore? https://github.com/rapidsai/cuml/blob/fd232a18f75c1ce3b2913fbfac93540828e4d72f/cpp/bench/sg/linkage.cu#L22

Reply:
Done.